boot/zephyr/nrf53_cpunet_ctl.c

```diff
@@ -0,0 +1,88 @@
+#include <zephyr.h>
+#include <hal/nrf_reset.h>
+#include <hal/nrf_spu.h>
+#include <dfu/pcd.h>
+#include <pm_config.h>
+#include "bootutil/bootutil_log.h"
+MCUBOOT_LOG_MODULE_DECLARE(mcuboot);
+
+int do_network_core_update(void *addr, void *src_addr, size_t len)
+{
+	struct pcd_cmd *rsp = (struct pcd_cmd *) PCD_RSP_ADDRESS;
+	if (addr == NULL) {
+		addr = (void *)PCD_CMD_ADDRESS;
+
+	}
+
+	struct pcd_cmd *cmd = (struct pcd_cmd *)addr;
+
+	rsp->magic = PCD_CMD_MAGIC_COPY;
+	cmd->magic = PCD_CMD_MAGIC_COPY;
+	cmd->src_addr = src_addr;
+	cmd->len = len;
+	cmd->offset = 0x10800;
```

Comment:
Where is this from? PM_nrf5340pdk_nrf5340_cpunet_APP?

Reply:
No it's PM_PROVISION_ADDRESS=0x1010000 - FLASH_START_ADDR